Source/Core/Core/PowerPC/Gekko.h

```diff
@@ -926,6 +926,12 @@ enum
   EXCEPTION_FAKE_MEMCHECK_HIT = 0x00000200,
 };
 
+enum CPUEmuFeatureFlags
```

Comment:
Given these are bitflags, this should probably be declared as:

```cpp
enum CPUEmuFeatureFlags : u32
```

so no weird funky signedness shenanigans can happen